pom.xml

```diff
@@ -6,13 +6,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.microsoft.azure.v3</groupId>
-  <artifactId>autorest-clientruntime-for-java</artifactId>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-common-for-java</artifactId>
```

Comment:
I would propose we call this `azure-common-parent`

Reply:
will call it `azure-common-parent`

Reply:
Let's just go with 1.0.0-SNAPSHOT. If we then want to do preview, we will [version it according to the Java spec](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html#sec-preview-releases) (so, 1.0.0-alpha1, 1.0.0-beta1, or 1.0.0-rc1).

Reply:
just pushed with 0.1.0, let me change it to 1.0.0-SNAPSHOT. Thanks for the reference.